Library/Source/NativeEngine.cpp

```diff
@@ -242,10 +245,17 @@ namespace babylon
 
         bgfx::setViewClear(0, BGFX_CLEAR_COLOR | BGFX_CLEAR_DEPTH, 0x443355FF, 1.0f, 0);
         bgfx::setViewRect(0, 0, 0, m_size.Width, m_size.Height);
-
         EngineDefiner::Define(env, this);
     }
 
+    void NativeEngine::Impl::Initialize(Napi::Env& env)
```

Comment:
I feel like I might have already asked this and just forgotten the answer, but why is this a separate method? Is there a circumstance under which we want to call InitializeRendering() but not Initialize()? Or is there an intention to add additional functionality to Initialize() that's unrelated to rendering?

Reply:
Moved init code in Initialize and updated comment.